code/go/internal/validator/test/packages/good/dataset/foo/_dev/test/pipeline/test-access-raw.log-expected.json

```diff
@@ -1,5 +1,5 @@
 {
-    "1expected": [
```

Comment:
Thanks for fixing!

Reply:
That one was actually on purpose... it would fail when somebody implements the validation :)